browser/package.json

```diff
@@ -24,19 +24,20 @@
     "classnames": "2.3.1",
     "file-saver": "1.3.8",
     "igniteui-dockmanager": "1.16.1",
-    "igniteui-react": "18.8.1",
-    "igniteui-react-charts": "18.8.1",
-    "igniteui-react-core": "18.8.1",
-    "igniteui-react-dashboards": "18.8.1",
-    "igniteui-react-datasources": "18.8.1",
-    "igniteui-react-excel": "18.8.1",
-    "igniteui-react-gauges": "18.8.1",
-    "igniteui-react-grids": "18.8.1",
-    "igniteui-react-inputs": "18.8.1",
-    "igniteui-react-layouts": "18.8.1",
-    "igniteui-react-maps": "18.8.1",
-    "igniteui-react-spreadsheet": "18.8.1",
-    "igniteui-react-spreadsheet-chart-adapter": "18.8.1",
+    "igniteui-react": "19.0.0-alpha.4",
+    "igniteui-react-charts": "19.0.0-alpha.0",
+    "igniteui-react-core": "19.0.0-alpha.0",
+    "igniteui-react-dashboards": "19.0.0-alpha.0",
+    "igniteui-react-datasources": "19.0.0-alpha.0",
+    "igniteui-react-excel": "19.0.0-alpha.0",
+    "igniteui-react-gauges": "19.0.0-alpha.0",
+    "igniteui-react-grids": "19.0.0-alpha.0",
+    "igniteui-react-data-grids": "19.0.0-alpha.0",
+    "igniteui-react-inputs": "19.0.0-alpha.0",
+    "igniteui-react-layouts": "19.0.0-alpha.0",
+    "igniteui-react-maps": "19.0.0-alpha.0",
+    "igniteui-react-spreadsheet": "19.0.0-alpha.0",
+    "igniteui-react-spreadsheet-chart-adapter": "19.0.0-alpha.0",
```

Comment:
@mddifilippo89 @gmurray81 
Is there a specific reason versions are locked here too? Do we delete the package lock in some step of the process? Just something that should probably be documented somewhere, because that's not what a normal `npm i <package>@next` would assign. At the very least I'd think `~19` would be an acceptable range.

Reply:
yes, we set specific version because we test samples with that version. This prevents installing minor versions that might have breaking API changes that would also break samples.

Reply:
Right, the React (and Angular) versioning based on the framework, thus breaking between 18.1 and 18.2 are possible. Still, in that example, using `~18.1.0` will only pick up patches of 18.1 and that's _if_ the lock file is removed or broken (otherwise `npm install` has been respecting it for a long time now). Anyway, I don't mind and will keep the deps versioning as it was :)